src/main/java/org/folio/qm/service/population/impl/Tag010FieldItemPopulationService.java

```diff
@@ -22,10 +22,12 @@
 public class Tag010FieldItemPopulationService extends FieldItemMarcPopulationService {
 
   private static final Pattern LCCN_FIELD_PATTERN = Pattern.compile("[$][abz].*$");
-  private static final Pattern LCCN_10_L_FIELD_PATTERN = Pattern.compile("\\d{10}");
-  private static final Pattern D_8 = Pattern.compile("\\d{8}");
+  private static final Pattern LCCN_10_DIGITS_ONLY_PATTERN = Pattern.compile("\\d{10}");
+  private static final Pattern LCCN_10_ONLY_LETTER_PREFIX_PATTERN = Pattern.compile("[a-z]\\d{10}");
+  private static final Pattern LCCN_8_DIGITS_ONLY_PATTERN = Pattern.compile("\\d{8}");
+  private static final Pattern LCCN_8_ONE_OR_TWO_LETTER_PREFIX_PATTERN = Pattern.compile("([a-z][ |a-z]? ?)(\\d{8})");
```

Comment:
Considering mod-record-specifications PR I would assume that pipe `|` is not a valid character, is it?